venues/ICLR.cc/2020/Conference/python/setup-stages.py

```diff
@@ -2,6 +2,120 @@
 import argparse
 import datetime
 
+remove_review_fields = ['confidence']
+additional_review_fields = {
+    'title': {
+        'order': 1,
+        'value-regex': 'Official Blind Review #[0-9]+',
+        'default': 'Official Blind Review #NUM',
+        'description': 'Please replace NUM with your AnonReviewer number (it is the number following "AnonReviewer" in your signatures below)',
+        'required': True
+    },
+    'review': {
+        'order': 2,
+        'value-regex': '[\\S\\s]{500,200000}',
+        'description': 'Provide your complete review here (500 - 200000 characters). For guidance in writing a good review, see this brief reviewer guide (https://iclr.cc/Conferences/2020/ReviewerGuide) with three key bullet points.',
+        'required': True
+    },
+    'rating': {
+        'order': 3,
+        'value-dropdown': [
+            '1: Reject',
+            '3: Weak Reject',
+            '6: Weak Accept',
+            '8: Accept'
+        ],
+        'required': True
+    },
+    'experience assessment': {
+        'order': 4,
+        'value-radio': [
+            'I have published in this field for several years.',
+            'I have published one or two papers in this area.',
+            'I have read many papers in this area.',
+            'I do not know much about this area.'
+        ],
+        'description': 'Please make a selection that represents your experience correctly',
+        'required': True
+    },
+    'review assessment: thoroughness in paper reading': {
```

Comment:
is it ok to create field name like this one? @zbialecki ?

Reply:
Is it the ":" that is troubling you? I can replace it with some other acceptable character. 
PCs wanted three questions in a category, and this was the most straight forward way that I could think of.

Reply:
I think it should be technically OK, but it might look a little weird in the form.

Reply:
This is how it shows up:

![Screen Shot 2019-10-03 at 12 56 43 PM](https://user-images.githubusercontent.com/13500158/66218728-c325df00-e697-11e9-9c4d-05059b0746bf.png)

Reply:
Actually, it looks pretty good to me

Reply:
I'm talking about spaces, we decided to use _ to represent spaces, but I'm not sure about having semicolons in the field names

Reply:
I see you updated the checkboxes to be radio buttons. It might be better to change the descriptions to something like the description for Experience Assessment.

Reply:
Good point @zbialecki. I'll make the change. Thanks.